templates/quickstart-bitbucket-dc.template.yaml

```diff
@@ -628,131 +672,110 @@ Resources:
-              content:
-                !Join
-                - "\n"
-                -
-                  - if [ -f ~/.bashrc ]; then
-                  -     . ~/.bashrc
-                  - fi
-                  - !Sub ["export CATALINA_OPTS=\"${CatalinaOpts}\"", CatalinaOpts: !Ref CatalinaOpts]
-              mode: '000644'
-              owner: atlbitbucket
-              group: atlbitbucket
-          commands:
-            010_add_nfs_mount:
-              command: !Sub
-                - "echo ${FileServerPrivateIp}:/media/atl/bitbucket/shared /media/atl/bitbucket/shared nfs lookupcache=pos,noatime,intr,rsize=32768,wsize=32768 0 0 >>/etc/fstab"
-                - FileServerPrivateIp: !GetAtt FileServer.PrivateIp
-              ignoreErrors: false
-            020_make_shared_home_dir:
-              command: mkdir -p /media/atl/bitbucket/shared
-              ignoreErrors: false
-            030_chown_shared_home_dir:
-              command: "chown atlbitbucket:atlbitbucket /media/atl/bitbucket /media/atl/bitbucket/shared"
-              ignoreErrors: false
-          services:
-            sysvinit:
-              cfn-hup:
-                enabled: true
-                ensureRunning: true
-                files: [/etc/cfn/cfn-hup.conf, /etc/cfn/hooks.d/cfn-auto-reloader.conf]
-              collectd:
-                !If [SetupCollectd, {enabled: true, ensureRunning: true}, !Ref "AWS::NoValue"]
-              rpcbind:
-                enabled: true
-                ensureRunning: true
-                packages:
-                  yum: [nfs-utils]
-        '2':
+
           commands:
-            040_mount_nfs:
-              command: "bash -c \"until mount /media/atl/bitbucket/shared; do echo 'Mount of shared volume failed trying again in 5 seconds.'; sleep 5; done\""
+            070_create_atl_dir:
+              test: "test ! -d /opt/atlassian/"
+              command: mkdir -p /opt/atlassian
               ignoreErrors: false
-          services:
-            sysvinit:
-              nfslock:
-                enabled: true
-                ensureRunning: true
-                packages:
-                  yum: [nfs-utils]
+            071_install_packages:
+              command: yum install -y git python-virtualenv
+              ignoreErrors: true
+            072_clone_atl_scripts:
+              test: "test ! -d /opt/atlassian/dc-deployments-automation/"
+              command: /opt/atlassian/bin/clone_deployment_repo
+              ignoreErrors: true
+            080_run_atl_init_node:
+              command: !Sub |
+                cd /opt/atlassian/dc-deployments-automation/ && ./bin/install-ansible && ./bin/ansible-with-atl-env inv/aws_node_local ${DeploymentAutomationPlaybook} /var/log/ansible-bootstrap.log
+              ignoreErrors: true
+
     Properties:
       AssociatePublicIpAddress: false
       BlockDeviceMappings:
-        - DeviceName: /dev/xvdf
+        - DeviceName: /dev/sdf
```

Comment:
Is this due to Amazon Linux 2? Is this also affected by the NVMe disks with different mapping? https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/nvme-ebs-volumes.html

Reply:
I tried using xvdf and had some issues, so just used sdf.